core/providers/PreloadServiceProvider.php

```diff
@@ -2,9 +2,9 @@
 
 namespace ImpressCMS\Core\Providers;
 
+use icms_preload_Handler as PreloadHandler;
```

Comment:
is this related with the aura session lib?

Reply:
sometimes with session lib same preloads was invoked few times and that was it because how preloads services previously was registered in container so this change kinda related